pkg/test/utils/subset.go

```diff
@@ -73,9 +73,12 @@ func IsSubset(expected, actual interface{}) error {
 			}
 
 			if err := IsSubset(iter.Value().Interface(), actualValue.Interface()); err != nil {
-				subsetErr := err.(*SubsetError)
-				subsetErr.AppendPath(iter.Key().String())
-				return subsetErr
+				subsetErr, ok := err.(*SubsetError)
```

Comment:
`IsSubset` seems to return only `SubsetError`s for now.

Reply:
True. Do you think we should change its signature to return `*SubsetError`s instead?